src/pages/workspace/withFullPolicy.js

```diff
@@ -60,6 +60,10 @@ const fullPolicyPropTypes = {
 
         /** A list of emails for the employees on the policy */
         employeeList: PropTypes.arrayOf(PropTypes.string),
+
+        /** A list of emails for the pendingInvitations to the policy */
+        // All invited emails will be inserted to pendingInvitations first, after success will be moved to employeeList
```

Comment:
I think it's best to keep the same comment format here by making a multi line comment using `/** */

Reply:
Okay